ts/features/itwallet/navigation/useItwLinkingOptions.tsx

```diff
@@ -12,6 +13,13 @@ import { ITW_ROUTES } from "./routes";
 export const useItwLinkingOptions = (): PathConfigMap<AppParamsList> => {
   const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
   const isItwEnabled = useIOSelector(isItwEnabledSelector);
+  const isMdlPresent = useIOSelector(itwHasMdlCredentialSelector);
+
+  const presentationRoute = isItwValid
+    ? isMdlPresent
+      ? ITW_ROUTES.PRESENTATION.CREDENTIAL_DETAIL
+      : ITW_ROUTES.ISSUANCE.CREDENTIAL_NOT_FOUND
+    : ITW_ROUTES.ISSUANCE.CREDENTIAL_ASYNC_FLOW_CONTINUATION;
```

Comment:
Is this deliberate to reuse the "activate wallet" logic within the async flow continuation screen? Because this screen  is not related to the credential detail/presentation.

Reply:
Yes as you guessed I reused the screen for the wallet activation part. I know it's not a super clean solution but in case we all agree I can create a new dedicated screen, the problem is I feel like I'm duplicating something already done.

Reply:
In my opinion we can reuse that screen, I just wanted to be sure of the reason.